es6-classes/student.js

```diff
@@ -0,0 +1,21 @@
+/* exported Student */
+
+
+class Student {
+  constructor(firstName, lastName, subject) {
+    this.firstName = firstName;
+    this.lastName = lastName;
+    this.subject = subject;
+  }
+}
+
+Student.prototype.getFullName = function () {
+  const { firstName, lastName } = this;
+  return `${firstName} ${lastName}`;
+};
+
+Student.prototype.getIntroduction = function () {
+  const { subject } = this;
+  const fullName = this.getFullName();
+  return `Hello, my name is ${fullName} and I am studying ${subject}.`;
+};
```

Comment:
These 2 methods should be moved inside of the class definition, same as you did with the methods from `Stopwatch`